mobile/lib/firebase_options.dart

```diff
@@ -0,0 +1,76 @@
+// File generated by FlutterFire CLI.
+// ignore_for_file: lines_longer_than_80_chars, avoid_classes_with_only_static_members
+import 'package:firebase_core/firebase_core.dart' show FirebaseOptions;
+import 'package:flutter/foundation.dart'
+    show defaultTargetPlatform, kIsWeb, TargetPlatform;
+
+/// Default [FirebaseOptions] for use with your Firebase apps.
+///
+/// Example:
+/// ```dart
+/// import 'firebase_options.dart';
+/// // ...
+/// await Firebase.initializeApp(
+///   options: DefaultFirebaseOptions.currentPlatform,
+/// );
+/// ```
+class DefaultFirebaseOptions {
+  static FirebaseOptions get currentPlatform {
+    if (kIsWeb) {
+      throw UnsupportedError(
+        'DefaultFirebaseOptions have not been configured for web - '
+        'you can reconfigure this by running the FlutterFire CLI again.',
+      );
+    }
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.android:
+        return android;
+      case TargetPlatform.iOS:
+        return ios;
+      case TargetPlatform.macOS:
+        return macos;
+      case TargetPlatform.windows:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for windows - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      case TargetPlatform.linux:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions have not been configured for linux - '
+          'you can reconfigure this by running the FlutterFire CLI again.',
+        );
+      default:
+        throw UnsupportedError(
+          'DefaultFirebaseOptions are not supported for this platform.',
+        );
+    }
+  }
+
+  static const FirebaseOptions android = FirebaseOptions(
+    apiKey: 'AIzaSyCDI7arcBp4uriwfLySgqrchkNVDVW94uk',
+    appId: '1:237191932527:android:651be56a933e8bda36bb9a',
+    messagingSenderId: '237191932527',
+    projectId: 'project-3195624368411891722',
+    storageBucket: 'project-3195624368411891722.appspot.com',
+  );
+
+  static const FirebaseOptions ios = FirebaseOptions(
+    apiKey: 'AIzaSyD1FiZUHkX-rdHUtAXU12ku2e7zmBHqLSQ',
+    appId: '1:237191932527:ios:ed5586aaa838be1e36bb9a',
+    messagingSenderId: '237191932527',
+    projectId: 'project-3195624368411891722',
+    storageBucket: 'project-3195624368411891722.appspot.com',
+    iosClientId: '237191932527-70uoogr06k3rrrmslr5u1r6i4st9ch39.apps.googleusercontent.com',
+    iosBundleId: 'finance.get10101.app',
+  );
```

Comment:
Do we need to differentiate between mainnet and regtest?

Reply:
Do you think it's worth differentiating between mainnet and regtest? I would rather assume that the tokens (or registration emails) should filter out our testing from other data, so we have clear data for analytics.

note that the file is autogenerated and should not be tempered with.